Build error (libgit2_clar & libcurl)

I haven't been able to build master for a few days now... I've used `make distcleanall` and even refreshed the whole repo directory at some point, but it always gets stuck. The problem is building libgit and curl, etc. Any hints as to what might be going wrong?

```
...
             ^
[ 22%] Linking C executable libgit2_clar
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `SSL_get_error@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `SSL_get_certificate@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `UI_method_get_reader@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `PEM_write_bio_X509@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `SSL_get_peer_certificate@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `SSL_CTX_load_verify_locations@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `SSL_CTX_use_PrivateKey@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `CRYPTO_free@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `SSL_CTX_use_certificate@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `SSL_CTX_set_msg_callback@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `ENGINE_free@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `ENGINE_get_next@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `UI_get0_user_data@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `SHA256_Init@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `X509_get_ext_d2i@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `PEM_read_X509@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `SSL_get1_session@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `SSL_write@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `UI_method_set_reader@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `MD5_Init@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `RAND_add@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `X509_EXTENSION_get_data@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `sk_num@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `SSL_CTX_add_client_CA@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `SSL_get_peer_cert_chain@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `i2t_ASN1_OBJECT@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `OCSP_single_get0_status@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `CONF_modules_load_file@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `ERR_get_error@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `SSL_CTX_set_srp_username@OPENSSL_1.0.1'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `PKCS12_PBE_add@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `UI_set_result@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `ENGINE_by_id@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `SSL_CTX_set_default_passwd_cb@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `X509_NAME_ENTRY_get_data@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `SSL_CIPHER_get_name@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `OCSP_resp_count@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `MD5_Update@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `X509_get_subject_name@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `ENGINE_init@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `BN_num_bits@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `ERR_clear_error@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `SSL_load_error_strings@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `SSL_version@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `SSL_CTX_set_default_passwd_cb_userdata@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `OCSP_RESPONSE_free@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `BIO_free@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `OCSP_response_status_str@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `UI_method_get_writer@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `DES_ecb_encrypt@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `X509_STORE_set_flags@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `DES_set_odd_parity@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `SSL_new@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `OCSP_basic_verify@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `ENGINE_get_first@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `X509_NAME_get_index_by_NID@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `DES_set_key@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `SHA256_Update@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `OPENSSL_add_all_algorithms_noconf@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `SSL_get0_alpn_selected@OPENSSL_1.0.2'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `ASN1_STRING_to_UTF8@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `UI_method_set_writer@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `RAND_egd@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `SSL_COMP_free_compression_methods@OPENSSL_1.0.2'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `ENGINE_set_default@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `ENGINE_cleanup@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `ERR_error_string@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `OCSP_resp_get0@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `SSL_get_shutdown@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `SSL_CTX_use_PrivateKey_file@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `X509_NAME_get_entry@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `SSL_set_connect_state@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `BIO_printf@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `ASN1_STRING_type@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `ASN1_INTEGER_get@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `SSL_library_init@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `RAND_status@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `SSL_get_current_cipher@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `UI_create_method@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `PKCS12_parse@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `X509_check_issued@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `SSL_CTX_set_cipher_list@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `OCSP_response_get1_basic@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `MD4_Final@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `MD4_Init@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `SSL_shutdown@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `SSL_connect@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `ERR_error_string_n@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `MD4_Update@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `i2a_ASN1_OBJECT@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `SSLv23_client_method@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `X509_EXTENSION_get_object@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `BIO_ctrl@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `d2i_OCSP_RESPONSE@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `SSL_CTX_use_certificate_chain_file@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `ASN1_STRING_length@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `BIO_new@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `BIO_puts@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `SSL_CTX_get_cert_store@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `SSLv3_client_method@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `sk_pop@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `i2d_X509_PUBKEY@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `OCSP_cert_status_str@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `ASN1_TIME_print@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `X509_NAME_print_ex@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `SSL_ctrl@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `PKCS12_free@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `SSL_set_session@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `RAND_load_file@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `sk_pop_free@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `d2i_PKCS12_fp@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `sk_value@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `SSL_CTX_check_private_key@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `ERR_peek_error@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `SSL_CTX_set_srp_password@OPENSSL_1.0.1'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `X509_load_crl_file@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `UI_method_get_opener@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `SHA256_Final@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `ASN1_STRING_print@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `OPENSSL_load_builtin_modules@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `RAND_bytes@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `SSL_CTX_new@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `ENGINE_load_private_key@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `UI_OpenSSL@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `SSLeay@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `OCSP_check_validity@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `ASN1_STRING_data@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `ENGINE_load_builtin_engines@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `CRYPTO_malloc@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `ENGINE_ctrl_cmd@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `X509_STORE_add_lookup@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `UI_get_string_type@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `EVP_cleanup@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `UI_get_input_flags@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `SSL_SESSION_free@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `X509_get_serialNumber@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `X509V3_EXT_print@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `ENGINE_finish@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `OCSP_BASICRESP_free@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `X509_get_issuer_name@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `SSL_CTX_set_next_proto_select_cb@OPENSSL_1.0.1'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `X509_get_pubkey@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `OCSP_response_status@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `SSL_free@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `UI_destroy_method@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `BIO_s_mem@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `OCSP_crl_reason_str@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `SSL_read@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `ERR_remove_thread_state@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `CRYPTO_cleanup_all_ex_data@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `SSL_get_privatekey@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `SSL_get_verify_result@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `ENGINE_ctrl@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `SSL_CTX_free@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `ERR_free_strings@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `EVP_PKEY_free@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `X509_verify_cert_error_string@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `CONF_modules_free@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `EVP_PKEY_copy_parameters@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `SSL_CTX_use_certificate_file@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `SSL_CTX_ctrl@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `UI_method_get_closer@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `SSL_pending@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `SSL_CTX_set_verify@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `BN_print@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `RAND_file_name@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `MD5_Final@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `UI_method_set_opener@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `GENERAL_NAMES_free@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `SSL_CTX_set_alpn_protos@OPENSSL_1.0.2'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `X509_free@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `UI_method_set_closer@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `X509_LOOKUP_file@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `SSL_set_fd@OPENSSL_1.0.0'
/home/aferris/julia-dev/usr/lib/libcurl.so: undefined reference to `ENGINE_get_id@OPENSSL_1.0.0'
collect2: error: ld returned 1 exit status
CMakeFiles/libgit2_clar.dir/build.make:12364: recipe for target 'libgit2_clar' failed
make[4]: *** [libgit2_clar] Error 1
CMakeFiles/Makefile2:104: recipe for target 'CMakeFiles/libgit2_clar.dir/all' failed
make[3]: *** [CMakeFiles/libgit2_clar.dir/all] Error 2
Makefile:138: recipe for target 'all' failed
make[2]: *** [all] Error 2
/home/aferris/julia-dev/deps/libgit2.mk:59: recipe for target 'build/libgit2-211e117a0590583a720c53172406f34186c543bd/libgit2.so' failed
make[1]: *** [build/libgit2-211e117a0590583a720c53172406f34186c543bd/libgit2.so] Error 2
Makefile:81: recipe for target 'julia-deps' failed
make: *** [julia-deps] Error 2
```

EDIT: this is Ubuntu 16.04 
```
aferris@aferris-desktop:~/julia-dev$ uname -a
Linux aferris-desktop 4.4.0-31-generic #50-Ubuntu SMP Wed Jul 13 00:07:12 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux
```

--- a/deps/curl.mk
+++ b/deps/curl.mk
@@ -16,7 +16,7 @@ $(SRCDIR)/srccache/curl-$(CURL_VER)/configure: $(SRCDIR)/srccache/curl-$(CURL_VE
 $(BUILDDIR)/curl-$(CURL_VER)/config.status: $(SRCDIR)/srccache/curl-$(CURL_VER)/configure
 	mkdir -p $(dir $@)
 	cd $(dir $@) && \
-	$< $(CONFIGURE_COMMON) --includedir=$(build_includedir) --with-mbedtls=$(build_shlibdir)/.. CFLAGS="$(CFLAGS) $(CURL_CFLAGS)" LDFLAGS="$(LDFLAGS) $(CURL_LDFLAGS)"
+	$< $(CONFIGURE_COMMON) --includedir=$(build_includedir) --without-ssl --with-mbedtls=$(build_prefix) CFLAGS="$(CFLAGS) $(CURL_CFLAGS)" LDFLAGS="$(LDFLAGS) $(CURL_LDFLAGS)"
 	touch -c $@
 $(CURL_SRC_TARGET): $(BUILDDIR)/curl-$(CURL_VER)/config.status
 	$(MAKE) -C $(dir $<) $(LIBTOOL_CCLD)

